website/source/docs/agent/options.html.md

```diff
@@ -457,6 +457,8 @@ will exit with an error at startup.
   the Web UI resources for Consul. This will automatically enable the Web UI. The directory must be
   readable to the agent. Starting with Consul version 0.7.0 and later, the Web UI assets are included in the binary so this flag is no longer necessary; specifying only the `-ui` flag is enough to enable the Web UI. Specifying both the '-ui' and '-ui-dir' flags will result in an error.
 
+* <a name="_ui_content_path"></a><a href="#_ui_content_path">`-ui-content-path`</a> - This flag provides the option to change the path the Web UI loads from. By default this will go to `/ui/`. Alphanumerics, -, and _, only, `/v1/` is not able to be as it would overwrite the API endpoint. 
```

Comment:
```suggestion
* <a name="_ui_content_path"></a><a href="#_ui_content_path">`-ui-content-path`</a> - This flag provides the option to change the path the Web UI loads from. By default this will go to `/ui/`. Alphanumerics, -, and _, only, `/v1/` is not allowed as it would overwrite the API endpoint. 
```